packages/modifycerttemplate/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=modifycerttemplate
+_pkgname=modifyCertTemplate
+pkgver=7.4c35708
+pkgrel=1
+pkgdesc='Aid operators in modifying ADCS certificate templates so that a created vulnerable state can be leveraged for privilege escalation.'
+arch=('any')
+groups=('blackarch' 'blackarch-windows')
+url='https://github.com/fortalice/modifyCertTemplate'
+license=('custom:unknown')
+depends=('python' 'python-ldap3' 'impacket')
+makedepends=('git')
+source=("git+https://github.com/fortalice/$_pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $_pkgname
+  echo $(git rev-list --count HEAD).$(git rev-parse --short HEAD)
+}
+
+package() {
+  cd $_pkgname
+
+  install -dm 755 "$pkgdir/usr/bin"
+  install -dm 755 "$pkgdir/usr/share/$pkgname"
+
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+
+  rm README.md .gitignore
```

Comment:
We can remove `[requirements.txt](https://github.com/fortalice/modifyCertTemplate/blob/master/requirements.txt)` too.

Reply:
Done